grails-app/services/org/bbop/apollo/PermissionService.groovy

```diff
@@ -517,11 +542,12 @@ class PermissionService {
             return false
         }
 
+        return isUserBetterOrEqualRank(user,permissionEnum)
         // if the rank required is less than administrator than ask if they are an administrator
-        if (PermissionEnum.ADMINISTRATE.rank < permissionEnum.rank) {
-            return isUserAdmin(user)
-        }
-        return true
+//        if (PermissionEnum.ADMINISTRATE.rank < permissionEnum.rank) {
+//            return isUserGlobalAdmin(user)
+//        }
+//        return true
     }
 
     Boolean hasPermissions(JSONObject jsonObject, PermissionEnum permissionEnum) {
```

Comment:
if permissionEnum = ADMINISTRATIVE, it will return false because ` if (!hasGlobalPermissions(jsonObject, permissionEnum)) {
            log.info("User lacks permissions ${permissionEnum.display}")
            return false
        }`

Reply:
@nathandunn I added the potential fix to this at #1873

Reply:
in favor of yours